src/psyclone/psyir/symbols/interfaces.py

```diff
@@ -110,14 +110,31 @@ class ImportInterface(SymbolInterface):
         from which the symbol is imported.
```

Comment:
Please add a line to the doc string saying that a Symbol may be renamed upon import and, if so, its original name in the Container may be specified using `orig_name`.

Reply:
Added.